front/app/containers/Admin/messaging/AutomatedEmails/ExampleModal/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from 'react';
+
+// i18n
+import T from 'components/T';
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../messages';
+
+// Components
+import Modal from 'components/UI/Modal';
+import EmptyState from './EmptyState';
+import ExampleFrame from './ExampleFrame';
+import { Button, Box, Text } from '@citizenlab/cl2-component-library';
+
+// Hooks
+import useCampaignExamples from 'api/campaign_examples/useCampaignExamples';
+import useCampaign from 'api/campaigns/useCampaign';
+
+/** Modulo function, since JS's native `%` remainder function works differently for negative numbers */
+const mod = (n: number, m: number) => ((n % m) + m) % m;
+
+const ExampleModal = ({
+  campaignId,
+  onClose,
+}: {
+  campaignId: string | null;
+  onClose: () => void;
+}) => {
+  const { data: examples, isLoading } = useCampaignExamples({ campaignId });
+  const { data: campaign } = useCampaign(campaignId);
+  const [selectedExampleIdx, setSelectedExampleIdx] = useState<number | null>(
+    null
+  );
+
+  useEffect(() => {
+    if (examples?.data?.length) {
+      setSelectedExampleIdx(0);
+    }
+  }, [examples]);
+
+  const changeSelectedExample = (offset: number) => {
+    setSelectedExampleIdx((idx) =>
+      mod((idx || 0) + offset, examples?.data.length || 1)
+    );
+  };
+
+  const selectedExample =
+    selectedExampleIdx !== null && examples?.data[selectedExampleIdx];
```

Comment:
```suggestion
  const selectedExample =
    selectedExampleIdx !== null ? examples?.data[selectedExampleIdx] : null;
```

Guess `null` makes more sense than `false` as potential type.